update raft log gc related tikv configurations (#7378)
pingcap/docs: tikv-configuration-file.md

+ Minimum value: `0`
+ Unit: MB|GB

> ### `raft-log-compact-sync-interval`

Reviewer: ```suggestion
### `raft-log-compact-sync-interval` <span class="version-mark">New in v5.3</span>
```

> + The time interval to compact unnecessary Raft logs.

Reviewer: ```suggestion
+ The time interval to compact unnecessary Raft logs
```

+ Default value: 3/4 of the Region size
+ Minimum value: greater than `0`

> ### `raft-log-reserve-max-ticks`

Reviewer: ```suggestion
### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
```

> + Old Raft logs could be reserved if `raft-log-gc-threshold` is not reached. GC them after `raft-log-reserve-max-ticks` ticks.

Reviewer: ```suggestion
+ If the maximum allowable count of residual Raft logs did not reach the value of `raft-log-gc-threshold`, old Raft logs are reserved. In this case, after `raft-log-reserve-max-ticks` is set, TiKV performs garbage collection (GC) to those Raft logs.
```

Author: Could you improve based on this new version?
"Performs garbage collection (GC) to residual Raft logs after `raft-log-reserve-max-ticks` ticks, even if their count doesn't reach the value of `raft-log-gc-threshold`."
And the "ticks" refers to elapsed time.

> + After the number of ticks set by this configuration item passed, even if the number of residual Raft logs does not reach the value set by `raft-log-gc-threshold`, TiKV still performs garbage collection (GC) to them.

Reviewer 2: ```suggestion
+ After the number of ticks set by this configuration item passes, even if the number of residual Raft logs does not reach the value set by `raft-log-gc-threshold`, TiKV still performs garbage collection (GC) to these logs.
```

+ Default value: `6`

> + Minimum value: greater than `0`

Reviewer 2: ```suggestion
+ Minimum value: greater than `0` 
```
